### ISSUE TITLE
在综合使用ernie3.0+crf的时候,loss.backward的时候报错

### ISSUE THREAD

**Issue by apexg** [PaddlePaddle/PaddleNLP#2371]
欢迎您反馈PaddleNLP使用问题，非常感谢您对PaddleNLP的贡献！
在留下您的问题时，辛苦您同步提供如下信息：
- 版本、环境信息
1）PaddleNLP和PaddlePaddle版本：PaddleNLP 2.3.1 PaddlePaddle2.2.2.post111
2）系统环境：请您描述系统类型，ubuntu18 python3.8

![6d15746b39fa605f0d941bf1ff6e9ec](https://user-images.githubusercontent.com/6739219/171126071-c07f4003-48b1-4804-a84e-2c6489073717.png)
模型信息:
![4adcf790b4081d05b8202db3a02ed98](https://user-images.githubusercontent.com/6739219/171126113-54d3b07e-db52-47f1-9f44-c8953970fb01.png)
![e2b58a63045b78293b2c83526424404](https://user-images.githubusercontent.com/6739219/171126147-17535e95-d35e-4fc7-877a-0da72883c30f.png)
